app/admin.py

```diff
@@ -108,6 +109,8 @@ def post(self):
                         'deactivation_message_html',
                         'keywords',
                         'launch_status',
+                        'admin_user_email',
```

Comment:
All repositories are usually owned by the same team, so I believe it would be better to make them global. We usually create one repository for each disaster. So it would be tedious to set the email address every time we create a repository. We may have different set of reviewers for each disaster, but we can handle it by specifying a mailing list here.